src/Compilers/Core/MSBuildTask/Sdk/Microsoft.Build.Tasks.CodeAnalysis.Sdk.csproj

```diff
@@ -22,5 +22,10 @@
     <InternalsVisibleTo Include="Microsoft.Build.Tasks.CodeAnalysis.Sdk.UnitTests" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)..\..\..\CSharp\csc\csc.rsp" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
>"PreserveNewest" [](http://example.com/codeflow?start=97&length=16)

Not sure how this option works. It feels like we should always overwrite unless exact match